components/webpack/webpack.config.js

```diff
@@ -11,6 +11,14 @@ const pathMap = require('./path-map')
 
 const tsConfigPath = path.join(process.env.ROOT_GEN_DIR, 'tsconfig-webpack.json')
 
+// OpenSSL 3 no longer supports the insecure md4 hash, but webpack < 6
```

Comment:
Actually I think that PR fix landed on webpack 5.54.0

Reply:
I see a `futureDefaults` flag that [selects xxhash64 instead](https://github.com/webpack/webpack/commits/da8e93af330739c332670ed7d040a648431703b4), but I don't think we set that? The linked issue says the don't intend to change the default because it breaks reproducibility.